### ISSUE TITLE
Feature Addition: Redesign the FAQ page nav bar with horizontal scroll

### ISSUE THREAD

**Issue by sanchitkalra** [Ashish-AVS/SWD-BPHC-Frontend#6]

**Comment by joshianirudh17** [comment]
I went through the page and it seems like horizontal scroll is required because the Material UI Tab titled "Math" is overflowing out of the area and is difficult to select. So, is it okay if I change the text and paddings so that there will not be a need of horizontal scroll at all?

**Comment by Ashish-AVS** [comment]
> I went through the page and it seems like horizontal scroll is required because the Material UI Tab titled "Math" is overflowing out of the area and is difficult to select. So, is it okay if I change the text and paddings so that there will not be a need of horizontal scroll at all?

Yes, feel free to make any changes that would eliminate the horizontal scrolling.
